gammapy/modeling/parameter.py

```diff
@@ -396,7 +415,7 @@ def __repr__(self):
         return (
             f"{self.__class__.__name__}(name={self.name!r}, value={self.value!r}, "
             f"factor={self.factor!r}, scale={self.scale!r}, unit={self.unit!r}, "
-            f"min={self.min!r}, max={self.max!r}, frozen={self.frozen!r}, id={hex(id(self))})"
+            f"min={self.min!r}, max={self.max!r}, frozen={self.frozen!r}, prior={self.prior!r}, id={hex(id(self))})"
         )
 
```

Comment:
@registerrier , in this PR should we implement here what is under review in the PR https://github.com/gammapy/gammapy/pull/3947?

Reply:
Yes, we should implement this. Maybe in a follow-up PR?